Fixed some errors for compatibility with Phonegap 2.5

mgraupner opened the pull request:
During the development of an application with PhoneGap 2.5 i tried to get the Push Plugin to work but it crashed the app. I fixed there errors in this Pull Requests but it probably will break compatibility with earlier versions of PhoneGap. So i guess if you think the changes are correct(they worked for me), you should make this the new master and push the former code into a branch or tag so people see that this only works for new versions.

Commit:
Changed JSON include to new CDVJSON version
Fixed tutorial error

mgcrea commented on README.md:
```diff
@@ -127,7 +127,7 @@ Using this plugin requires [Cordova Android](https://github.com/apache/incubator
         // {"type":"7","pushBadge":"1","pushSound":"1","enabled":"1","deviceToken":"blablahblah","pushAlert":"1"}
         console.warn('registerDevice:%o', status);
-        navigator.notification.alert(JSON.stringify(['registerDevice', status]));
+        navigator.notification.alert(JSON.stringify(event.notification.aps.alert));
```
I don't get this edit. There is no `event` var defined.

kaansoral commented:
Thanks a lot for sharing the code segment, it helped a lot :)
I don't code in a controlled experiment manner, however navigator.notification.alert might be depreciated, didn't turn back and check whether it was the source of the problems, however I was getting 'not an object' JS errors, so I alert(JSON.stringify(status))'d and got the actual response after hours of xcode certificate debugging and just close the macbook after that

Commit:
Fixed wrong example code

mgraupner commented:
I changed the wrong position of the JS Code by mistake, i fixed this in the last commit. Otherwise one would get an `TypeError: JSON.stringify cannot serialize cyclic structures` error.

mgcrea commented:
Should be fixed by #13
